apps/financial-aid/web-veita/src/components/MunicipalityAdminSettings/MunicipalityAdminSettings.tsx

```diff
@@ -37,6 +40,8 @@ const MunicipalityAdminSettings = ({ currentMunicipality }: Props) => {
 
   const [hasNavError, setHasNavError] = useState(false)
   const [hasAidError, setHasAidError] = useState(false)
+  const [hasDecemberCompensationError, setHasDecemberCompensationError] =
```

Comment:
Probably not part of this PR but I would consider in the near future to have the error logic in a separate file and import it.